dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/runner/TaskExecuteThread.java

```diff
@@ -256,4 +272,60 @@ private void downloadResource(String execLocalPath,
             }
         }
     }
+
+    /**
+     * delay execution if needed
+     */
+    private void delayExecutionIfNeeded() {
+        long remainTime = DateUtils.getRemainTime(taskExecutionContext.getFirstSubmitTime(),
+                taskExecutionContext.getDelayTime() * 60L);
+        logger.info("delay execution time: {} s", remainTime < 0 ? 0 : remainTime);
+        if (remainTime > 0) {
+            try {
+                Thread.sleep(remainTime * Constants.SLEEP_TIME_MILLIS);
+            } catch (Exception e) {
+                logger.error("exception", e);
+                logger.error("delay task execution failure, the task will be executed directly." +
```

Comment:
```suggestion
                logger.error("exception", e);
                logger.error("delay task execution failure, the task will be executed directly." +
```
Hi,
It will be better to combine these error log to one error log.